css/liz-wheeler.css

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
you created a new CSS file, which is good in the future you will be dividing css files but in this instance i may have been unnecessary.